### PR TITLE
voltLib: parsing VOLT files

### PR DESCRIPTION
**Pull request by moyogo**
This add fontTools.voltLib which much like feaLib parses OTL definitions but from VOLT files.
I haven’t had time to implement building yet, so merging can let others help.

### CODE REVIEW

File: Lib/fontTools/voltLib/ast.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
+from __future__ import print_function, division, absolute_import
+from __future__ import unicode_literals
+import fontTools.feaLib.ast as ast
```
Comment (brawer) [inline Lib/fontTools/voltLib/ast.py]: This dependency seems a bit weird; could voltLib define its own syntax trees?

File: Lib/fontTools/voltLib/parser.py [hunk 2]
```diff
@@ -0,0 +1,633 @@
+from __future__ import (
+    print_function, division, absolute_import, unicode_literals)
+from collections import OrderedDict
+import fontTools.voltLib.ast as ast
+import fontTools.feaLib.parser as parser
```
Comment (brawer) [inline Lib/fontTools/voltLib/parser.py]: Why does voltLib use the feaLib parser? The dependency on feaLib.parser.SymbolTable seems strange — doesn’t voltLib needs its own SymbolTable?

**Comment by anthrotype** [comment]
Awesome! Thank you Denis 👍

**Comment by moyogo** [comment]
@brawer oh there’s another commit right after you merged where I also remove the import from feaLib.parser. I cherry-picked it to master.

**Comment by brawer** [comment]
Thank you!

**Comment by behdad** [comment]
Evil!

**Pull request merged**